src/phel/core.phel

```diff
@@ -1269,8 +1269,8 @@ returns 1. If `xs` has one value, returns the reciprocal of x."
   The resulting string is returned."
   [& xs]
   (let [len (count xs)
-        printer (php/new Printer)
-        pp |(php/-> printer (print $ false))]
+        printer (php/new Printer false)
```

Comment:
@jenshaase is it possible to call a static method from here? I would like to do something like:
```
printer (php/-> Printer::nonReadable())
```
But I don't know if that's even possible. That's why I let the constructor public... but I would say, ideally it should be private, so you can create a Printer only by one of its two named constructors.

Anyway, independently of the proposal and design itself, would it possible?

Reply:
`(php/:: Printer (nonReadable))`